pounders/m/clean_up_before_return.m

```diff
@@ -0,0 +1,25 @@
+% This function is called to cleanup X and F, set the exit value, and display
+% reason for exiting
+
+% flag    [dbl] Termination criteria flag:
+%               = 0 normal termination because of grad,
+%               > 0 exceeded nfmax evals,   flag = norm of grad at final X
+%               = -1 if input was fatally incorrect (error message shown)
+%               = -2 model failure
+%               = -3 error from TRSP Solver or if a NaN was encountered
```

Comment:
This is different from the flag documentation in `pounders.m`.  Which is correct?

Reply:
As you pointed out, removing redundant documentation helped remove this issue.